zebra/interface.c

```diff
@@ -770,7 +770,17 @@ void if_nbr_ipv6ll_to_ipv4ll_neigh_update(struct interface *ifp,
 	inet_pton(AF_INET, buf, &ipv4_ll);
 
 	ipv6_ll_address_to_mac(address, (u_char *)mac);
-	kernel_neigh_update(add, ifp->ifindex, ipv4_ll.s_addr, mac, 6);
+
+	/*
+	* Remove existed arp record for the interface as netlink
```

Comment:
indent this comment to line up with the * on the first line and we'll be good to go